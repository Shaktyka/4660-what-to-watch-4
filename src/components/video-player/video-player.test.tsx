import * as React from 'react';
import * as renderer from 'react-test-renderer';

import VideoPlayer from './video-player';

describe(`VideoPlayer rendering`, () => {

  it(`VideoPlayer renders correctly`, () => {

    const tree = renderer
      .create(
          <VideoPlayer>
            <video />
          </VideoPlayer>, {
            createNodeMock: () => {
              return {};
            }
          }
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
