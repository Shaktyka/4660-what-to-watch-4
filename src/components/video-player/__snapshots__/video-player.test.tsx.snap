// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer rendering VideoPlayer renders correctly 1`] = `<video />`;
